packages/manager/modules/sign-up/src/form/form.controller.js

```diff
@@ -110,45 +110,43 @@ export default class SignUpFormCtrl {
       });
   }
 
-  initModel() {
+  /**
+   *  Define the model with the values from the response of GET /me.
+   *  Be sure to take only the attributes from the available post params of POST /newAccount/rules
+   *  to avoid errors on POST.
+   */
+  initModel(postParams) {
     this.model = {};
-    // define model from GET /me
-    const meCopy = Object.assign({}, this.me);
-    // and remove some unecessary attributes
-    delete meCopy.state;
-    delete meCopy.currency;
-    delete meCopy.customerCode;
-    delete meCopy.nichandle;
-    delete meCopy.model;
-    // define model properties
-    Object.keys(meCopy).forEach((key) => {
-      Object.defineProperty(this.model, `$${key}`, {
+
+    postParams.forEach(({ name }) => {
+      Object.defineProperty(this.model, `$${name}`, {
```

Comment:
No need ;-)

Reply:
Sorry, my question was, why having two chars `$` ?

Reply:
Ho, I see.
To avoid `$scope.$watch`, I us getter and setter to simulate it. As I need a value to be accessed/changed, I use a kind of `private` attributes which I set as not enumerable. The `public` attributes are my getters and setters.